docs/src/main/java/org/geotools/graph/GraphExamples.java

```diff
@@ -45,13 +43,7 @@ void graphExample() throws Exception {
         final LineGraphGenerator generator = new BasicLineGraphGenerator();
         SimpleFeatureCollection fc = featureSource.getFeatures();
 
-        fc.accepts(
-                new FeatureVisitor() {
-                    public void visit(Feature feature) {
-                        generator.add(feature);
-                    }
-                },
-                null);
+        fc.accepts(feature -> generator.add(feature), null);
```

Comment:
all of these feature visitor ones are great.

Reply:
Yep, they are the "low hanging fruit" of this PR indeed